### PR TITLE
fixed error of bitbucket in struct

### PR DESCRIPTION
**Pull request by nik0811**
fixed error of bitbucket while parsing the callback response
Not able to parse email properly so login was failling now fixed

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/markbates/goth
+module github.com/nik0811/goth
```
Comment (techknowlogick) [inline go.mod:1]: Thanks for PR, are you able to revert this change?

Comment (nik0811) [reply go.mod]: @techknowlogick  yes reverted,

**Comment by techknowlogick** [comment]
Thanks again :) I've just merged these changes.
